osu.Game/Skinning/BeatmapSkinExtensions.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Beatmaps;
+
+namespace osu.Game.Skinning
+{
+    public static class BeatmapSkinExtensions
+    {
+        public static SkinInfo CreateSkinInfo(BeatmapInfo beatmap) => new SkinInfo
```

Comment:
What's the intention of this? Is it used somewhere else? Even in the follow-up PR it's only called in one spot.

Reply:
No longer necessary now, missed it as well.